library_generation/generate_library_integration_test.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+
+set -xeo pipefail
+script_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source $script_dir/utilities.sh
+
+# This script is used to test the result of `generate_library.sh` against generated
+# source code in googleapis-gen repository.
+# Specifically, this script will do
+# 1. checkout the master branch of googleapis/google and WORKSPACE
+# 2. parse version of gapic-generator-java, protobuf and grpc from WORKSPACE
+# 3. generate a library with given proto_path and destination_path by invoking
+#    `generate_library.sh`. GAPIC options to generate a library will be parsed
+#    from proto_path/BUILD.bazel.
+# 4. checkout the master branch googleapis-gen repository and compare the result.
+
+# defaults
+googleapis_gen_url="git@github.com:googleapis/googleapis-gen.git"
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+case $key in
+    -p|--proto_path)
```

Comment:
nit: the indention of this line should be 2 whitespace, rather than 4.

Reply:
this was the wrong file I brought when merging `main`. The file under `library_generation/test` has the correct indentation